### PR TITLE
i18n support and Japanese translation

### PR DESCRIPTION
**Pull request by japonophile**
Added i18n initialization and implemented Japanese translation for kano-apps.
The UI is correctly displayed in Japanese, but each app description (fetched from .app files) is still in English only.

### CODE REVIEW

**Comment by alex5imon** [comment]
😍

**Comment by Ealdwulf** [comment]
Just to let you know that we haven't forgotten about these - This one looks pretty safe to me. However when I tried it nothing came up in Japanese (I know the descriptions don't, but I didn't find any Japanese). It's probably something trivial, I'll have another look soon.

**Comment by japonophile** [comment]
Great to hear back from you!

Here are the quick steps I follow to "japanify" my Kano:

```$ sudo dpkg-reconfigure locales```
Choose:
en_GB.UTF-8 UTF-8
ja_JP.EUC-JP EUC-JP
ja_JP.UTF-8 UTF-8 (*)

```$ apt-get install ttf-kochi-gothic xfonts-intl-japanese xfonts-intl-japanese-big xfonts-kaname```

I guess you may have missed that last step.
If the proper fonts are not available, you won't see anything.

**Comment by japonophile** [comment]
Another thing I noticed, if you are installing ```kano-apps``` through ```setup.py```, is that it will not copy the ```kano-apps``` script at the correct location.  To test on my Kano, I just manually copied kano-apps to ```/usr/bin```.

**Comment by Ealdwulf** [comment]
Thanks @japonophile . The problem I was having was that the debian/install file was missing the po files: https://github.com/KanoComputing/kano-apps/commit/c44fd0675846be8d35fb20f0c78325d3b35825b8
With that fixed it displays the Japanese fine. We are doing a  release right now, should be able to merge this afterwards.

File: kano_apps/AppManage.py [hunk 1]
```diff
@@ -67,7 +67,7 @@ def uninstall_packages(app, sudo_pwd=None):
 
     pkgs = " ".join(app["packages"])
 
-    cmd = "rxvt -title 'Uninstalling {}' -e bash -c ".format(app["title"])
+    cmd = ("rxvt -title " + _("'Uninstalling {}'") + " -e bash -c ").format(app["title"])
```
Comment (Ealdwulf) [inline kano_apps/AppManage.py]: There is a problem here, in that os.system expects ordinary strings, so we need to do `os.system(cmd.encode('utf8'))` below, and similarly elsewhere in this file. I've made this change locally.

**Comment by Ealdwulf** [comment]
Hi @japonophile , I'm going to formally close this one, bue actually I've opened a new one which incorporates your commits here: https://github.com/KanoComputing/kano-apps/pull/77
Thanks!

**Pull request merged**